app/components/common/Footer.vue

```diff
@@ -102,7 +102,7 @@ export default Vue.extend({
             items.HOC,
             items.GRANTS,
             items.PRIVACY,
-            items.CODEQUEST,
+            // items.CODEQUEST,
```

Comment:
always delete instead of comment. We can always go to git history and get deleted code back.